wordpress-mobile/WordPress-Android
# Issue/7540 followed blog notifications

## ~~Do Not Merge~~
~~This is dependent upon https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/774.  Once that is merged, the FluxC commit hash will need to be updated and these changes will be ready to merge following the code review.~~

### Fix
This is only ***Part 1 of 2*** for the [Notification for Followed Blogs issue](https://github.com/wordpress-mobile/WordPress-Android/issues/7540).  I divided the issue into two parts so the pull requests aren't too big.

Add followed blogs to Notification Settings allowing management of notifications for new post and comments.  See the screenshots below for illustration.

![notifications_settings_list](https://user-images.githubusercontent.com/3827611/38445070-2b1d3960-39af-11e8-9fcd-4a716fc73f4e.png)

![notifications_settings_dialog_collapsed](https://user-images.githubusercontent.com/3827611/38445073-2d490728-39af-11e8-9d30-afe0b1a88422.png)

![notifications_settings_dialog_expanded](https://user-images.githubusercontent.com/3827611/38445075-2f2fd332-39af-11e8-8e60-5ddd7d125009.png)

The `RadioGroup` in `followed_sites_dialog.xml` uses the `android:layoutDirection="rtl"` attribute in order to display the radio buttons to the right of the text as shown in the screenshots above.  Unfortunately, the `layoutDirection` attribute is not available on API 16 and below.  Consequently, the layouts will be different on API 16 compared to later versions.  See the screenshots below for illustration.

![notification_settings_followed_dialog_api](https://user-images.githubusercontent.com/3827611/38444253-da3ccf04-39ab-11e8-8584-7b989357c060.png)

### Test
1. Go to ***Me*** tab.
2. Tap ***Notification Settings*** item.
3. Notice ***Followed Sites*** section.
4. Tap site under ***Followed Sites*** section.
5. Edit notifications settings in dialog.
6. Tap ***OK*** button.
7. Tap site from Step 5.
8. Notice notifications settings are saved.
9. Edit notifications settings in dialog.
10. Tap ***Cancel*** button.
11. Tap site from Step 5.
12. Notice notifications settings are not saved.
